api/src/test/java/uk/gov/dhsc/htbhf/claimant/message/processor/MakePaymentMessageProcessorTest.java

```diff
@@ -50,6 +60,26 @@ void shouldProcessMessage() {
         verify(paymentService).makePaymentForCycle(paymentCycle, claim.getCardAccountId());
     }
 
+    @Test
+    void shouldTriggerPaymentEmailAfterSuccessfulPayment() {
```

Comment:
Should the message testing be simply added to `shouldProcessMessage`, would save some duplicate setup code and its just a part of the normal message processing.

Reply:
Done